scripts/convert-dlrs-rules.apex

```diff
@@ -34,6 +30,37 @@ for (dlrs__LookupRollupSummary2__mdt dlrsRule : dlrs__LookupRollupSummary2__mdt.
       operation = dlrsRule.dlrs__AggregateOperation__c;
     }
   }
+   if(invalidChildren.contains(dlrsRule.dlrs__ChildObject__c)){
+    //we will debug a list of unmigrateable rules to assist with the creation of the flow actions
+    Map<String,String> unmigratableRule = new Map<String,String>();
+    if(dlrsRule.dlrs__RelationshipCriteria__c != null){
+      unmigratableRule.put('SOQL Where Clause To Exclude Calc Items',dlrsRule.dlrs__RelationshipCriteria__c);
+    }
+    unmigratableRule.put('Records to rollup','Same as Prior records to rollup');
+    unmigratableRule.put('Prior records to rollup','provide the collection of child records (assuming rollup not from parent record else collection of parent records)');
+    if(dlrsRule.dlrs__FieldToOrderBy__c != null){
+    unmigratableRule.put('Order By (First/Last)',dlrsRule.dlrs__FieldToOrderBy__c);
+    }
+    if(operation.startsWith('CONCAT')){
+      unmigratableRule.put('Concat Delimiter',dlrsRule.dlrs__ConcatenateDelimiter__c );
+    }
+    unmigratableRule.put('Rollup target\'s SObject Name',dlrsRule.dlrs__ParentObject__c);    
+    unmigratableRule.put('Rollup Operation',operation.toUpperCase());
+    unmigratableRule.put('Rollup Object Field',dlrsRule.dlrs__AggregateResultField__c);   
+    unmigratableRule.put('Rollup Context','INSERT / UPDATE / UPSERT / DELETE: see documentation');
+    unmigratableRule.put('Lookup Field On Rollup Object','Id');
+    unmigratableRule.put('Lookup Field On Calc Item',dlrsRule.dlrs__RelationshipField__c);
+    unmigratableRule.put('Calc Item Rollup Field',dlrsRule.dlrs__FieldToAggregate__c);
+    unmigratableRule.put('Object for \"Prior records to rollup\"',dlrsRule.dlrs__ChildObject__c);
+    unmigratableRule.put('Action label',(String) dlrsRule.get('MasterLabel'));
+    unmigrateableRules.add(unmigratableRule);
+    continue;
+   }
+  Metadata.CustomMetadata customMetadata = new Metadata.CustomMetadata();
```

Comment:
these lines shouldn't be part of the changes - they should remain where they were originally, and your changes should add new lines below them

Reply:
I figured it was sensible to not bother starting the metadata variable if the rule wasn't going to be migrated?